### PR TITLE
Document missing Allocation Service helm variables

### LINKED ISSUE #2987: Document missing Allocation Service helm variables.
**What happened**:

We wanted to change the port name of the allocation service, but couldn't find a way! On inspection this is a configurable helm value, but it was not documented! In fact, there are several items not documented.

**What you expected to happen**:

Review:

https://github.com/googleforgames/agones/blob/bd6cc05b9ee6b4f2eb66186a49d4b665632638af/install/helm/agones/templates/service/allocation.yaml#L37-L64

And ensure everything is documented in the helm config docs.

**How to reproduce it (as minimally and precisely as possible)**:

Read:
https://agones.dev/site/docs/installation/install-agones/helm/#configuration

And see what is missing, for example `agones.allocator.service.http.portName` is missing.

**Anything else we need to know?**:

N/A

**Environment**:
- Agones version:  dev
- Kubernetes version (use `kubectl version`): N/A
- Cloud provider or hardware configuration:  N/A
- Install method (yaml/helm): helm
- Troubleshooting guide log(s): N/A
- Others: N/A

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix

/kind cleanup

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Closes #2987

**Special notes for your reviewer**:

### CODE REVIEW

File: site/content/en/docs/Installation/Install Agones/helm.md [hunk 1]
```diff
@@ -177,14 +177,17 @@ The following tables lists the configurable parameters of the Agones chart and t
...
 | `agones.allocator.service.name`                          | Service name for the allocator                                                                                                                                                                                          | `agones-allocator`                 |
 | `agones.allocator.service.serviceType`                   | The [Service Type][service] of the HTTP Service                                                                                                                                                                         | `LoadBalancer`                     |
+| `agones.allocator.service.http.nodePort`, `agones.allocator.service.grpc.nodePort`                   | The [NodePort] serves as the external entry point for incoming requests for your app                                                                                                                                                                         | `30000-32767`                     |
```
Comment (markmandel) [inline site/content/en/docs/Installation/Install Agones/helm.md:180]: Let's not combine the two, but have separate rows for each please.

We should also make clear - This parameter is only used if the ServiceType is set to "NodePort".

Thanks!

Comment (Kalaiselvi84) [reply site/content/en/docs/Installation/Install Agones/helm.md]: These two parameters are added in separate rows. Thanks!

File: site/content/en/docs/Installation/Install Agones/helm.md [hunk 2]
```diff
@@ -177,14 +177,18 @@ The following tables lists the configurable parameters of the Agones chart and t
...
 | `agones.allocator.service.name`                          | Service name for the allocator                                                                                                                                                                                          | `agones-allocator`                 |
 | `agones.allocator.service.serviceType`                   | The [Service Type][service] of the HTTP Service                                                                                                                                                                         | `LoadBalancer`                     |
+| `agones.allocator.service.http.nodePort`                   | The NodePort parameter is used if the ServiceType is set to "NodePort" and it serves as the external entry point for the incoming requests                                                                                                                                                                         | `30000-32767`                     |
```
Comment (markmandel) [inline site/content/en/docs/Installation/Install Agones/helm.md:180]:
```suggestion
| `agones.allocator.service.http.nodePort`                   | If the ServiceType is set to "NodePort",  this is the NodePort that the allocator http service is exposed on.                                                                                                                                                                      | `30000-32767`                     |
```

File: site/content/en/docs/Installation/Install Agones/helm.md [hunk 3]
```diff
@@ -177,14 +177,18 @@ The following tables lists the configurable parameters of the Agones chart and t
...
 | `agones.allocator.service.grpc.port`                     | The port that is exposed externally by the [allocator service][allocator] for [gRPC requests][grpc-requests]                                                                                                            | `443`                              |
+| `agones.allocator.service.grpc.portName`                     | The name of exposed port                                                                                                            | ``                              |
+| `agones.allocator.service.grpc.nodePort`                   | The NodePort parameter is used if the ServiceType is set to "NodePort" and it serves as the external entry point for the incoming requests                                                                                                                                                                          | `30000-32767`                     |
```
Comment (markmandel) [inline site/content/en/docs/Installation/Install Agones/helm.md:191]:
```suggestion
| `agones.allocator.service.grpc.nodePort`                   | If the ServiceType is set to "NodePort",  this is the NodePort that the allocator gRPC service is exposed on.                                                                                                                                                                  | `30000-32767`                     |
```

**Review by markmandel — COMMENTED** [review]
Couple of minor tweaks, and I think this is good to go (but we should also check the preview, just in case)

File: site/content/en/docs/Installation/Install Agones/helm.md (hunk 2 above)
Comment (Kalaiselvi84) [reply site/content/en/docs/Installation/Install Agones/helm.md]: Modified it.

**Comment by markmandel** [comment]
Sweet. Will wait on preview and if all looks good approve and merge 👍🏻

**Review by markmandel — APPROVED** [review]

**Pull request merged**